docs/search/search_for_content.md

```diff
@@ -3,26 +3,45 @@ description: Search through all content in the system and filter the results by
 ---
 
 # Search for content
+
 You can enter a search keyword and the application checks all the searchable Fields and returns a list of search results that contain the keyword.
 
+TODO: Update following screenshot to have the sorting menu visible and several translations.
+
 ![Basic Search](img/basic_search.png)
 
-You can also select a language to search in.
-The results will contain Content items that are translated into this language.
+Everywhere in the Back Office, completion suggestions are proposed while typing in the search box.
+If a suggestion is what you're looking for, click on it in the suggestion drop-down.
+If no suggestion correspond to your search, press Enter key or click "View all results".
```

Comment:
TODO: Add keyboard navigation, up & down

Reply:
and `ctrl + /` (win, linux) | `cmd + /` (mac) shortcats to active search

Reply:
Done in e7ee4eada53c507440bf465a76bbff54705194e2